docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst

```diff
@@ -126,3 +128,55 @@ cache:
             temporary-image \
             docker://$image
       scp temporary-image mycluster:$singularity_image_root/$image
+
+.. _manage-singularity-cache:
+
+********************************************************************************
+ Managing the Singularity Image Cache using the manage-singularity-cache script
+********************************************************************************
+
+A convenience script, ``/etc/launcher/scripts/manage-singularity-cache``,is provided by the HPC
+launcher installation to simplfiy the management of the Singularity image cache. The script
+simplifies the management of the Singularity image cache directory content and helps ensure proper
+name, placement, and permissions of content added to the cache. Adding container images to the
+Singularity image cache avoids the overhead of downloading of the images and allows for sharing of
+images between multiple users. It provides the following features:
+
+   -  Download the Determined default cuda, cpu, or rocm environment images
+   -  Download an arbitrary docker image reference
+   -  Copy a local Singularity image file into the cache
+   -  List the currently available images in the cache
+
+If your system has internet access, you can download images directly into the cache. Use the
+``--cuda``, ``--cpu``, or ``--rocm`` options to download the current default CUDA, CPU, or ROCM
+environment container image into the cache. For example, to load the default CUDA container image,
```

Comment:
Does "to load" need to be "to download"?

Reply:
Changed it.  Though it does both, but more consistent to say it the same between the sentences.